### DISCUSSION TITLE
new internal Confluence site with SEB

### DISCUSSION THREAD

**Discussion by ppas0** [SafeExamBrowser/seb-win-refactoring#934] [General]
Hello, i have a question.. is it possible to integrate/ start the SEB with an internal Confluence site?
Thanks in Advance

**Comment by dbuechel** [comment]
Yes, as long as the computer has access to the internal server, then you'd simply need to set the start URL accordingly in the SEB configuration tool (see https://safeexambrowser.org/windows/win_usermanual_en.html#GeneralPane).
